src/resolvers/Mutation/createMember.ts

```diff
@@ -0,0 +1,114 @@
+import { MutationResolvers } from "../../types/generatedGraphQLTypes";
+import { errors, requestContext } from "../../libraries";
+import { User, Organization } from "../../models";
+import { superAdminCheck } from "../../utilities";
+import {
+  ORGANIZATION_NOT_FOUND_ERROR,
+  USER_NOT_FOUND_ERROR,
+  MEMBER_NOT_FOUND_ERROR,
+} from "../../constants";
+/**
+ * This function enables to add a member.
+ * @param _parent - parent of current request
+ * @param args - payload provided with the request
+ * @param context - context of entire application
+ * @remarks The following checks are done:
+ * 1. Checks whether current user making the request is an superAdmin
+ * 2. If the organization exists
+ * 3. Checks whether curent user exists.
+ * 4. Checks whether user with _id === args.data.userId is already an member of organization..
+ * @returns Organization.
+ */
+export const createMember: MutationResolvers["createMember"] = async (
+  _parent,
+  args,
+  context
+) => {
+  // Checks whether the current user is a superAdmin
+  const currentUser = await User.findById({
```

Comment:
Make use of `findOne`, usage of multiple methods that do the same thing makes code inconsistent.